budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/BudgeteerSession.java

```diff
@@ -13,6 +13,8 @@
     //This makes them persistent when switching projects
     private HashMap<Long, BudgetTagFilter> budgetFilter = new HashMap<>();
 
+    private Long budgetRemainingFilter = 0L;
```

Comment:
The name is somewhat misleading. This is not a filter to itself, but rather a value used by the filter

Reply:
This has been fixed ;)